gromacs/testing.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2009 Oliver Beckstein <orbeckst@gmail.com>
+# Released under the GNU Public License 3 (or higher, your choice)
+# See the file COPYING for details.
+
+import os, inspect
+
+def get_data_path(f=''):
+    """Return the path of a data file, these are relative to the current test
+    directory.
+    """
+    # get our callers file
+    _, filename, _, _, _, _ = inspect.getouterframes(inspect.currentframe())[1]
```

Comment:
Why are they not using the [setuptools resource manager](https://pythonhosted.org/setuptools/pkg_resources.html#resourcemanager-api)? (That's what we're using in MDAnalysis to be sure that we get exactly our datafiles.) I know that you complained about that you have to add files in various places but isn't it cleaner to specifically [extract](https://pythonhosted.org/setuptools/pkg_resources.html#resource-extraction) a file than just hoping that you can find it somewhere under the directory where the test file lives? And it doesn't work with zipped eggs or possibly other fancy formats. I assume that the pandas guys do know what they are doing but it is not obvious to me why. I think its fugly.

That said... better any tests than no tests so if I didn't sway you into at least looking at the links then go ahead and we'll sort out the dead later.